app/views/shared/_review.html.erb

```diff
@@ -1,12 +1,12 @@
 <div class="alert alert-info">
   <% if @resource.latest_status_action(Action::REQUEST_REVIEW) %>
-    <p><%= @resource.latest_status_action(Action::REQUEST_REVIEW).requester.name %> has requested this edition be reviewed.</p>
     <% if @resource.latest_status_action.requester == current_user %>
-      <p>That&rsquo;s you! You'll get an email when it has been reviewed.</p>
+      <p>You’ve sent this edition to be reviewed.</p>
       <% if skip_review? %>
         <p><%= build_review_button(@resource, "skip_review", "Skip review").html_safe %></p>
       <% end %>
     <% else %>
+    <p><%= @resource.latest_status_action(Action::REQUEST_REVIEW).requester.name %> has sent this edition to be reviewed.</p>
```

Comment:
Code formatting nitpick: this line should be indented 2 spaces more now that it's inside the "else" block.

```suggestion
      <p><%= @resource.latest_status_action(Action::REQUEST_REVIEW).requester.name %> has sent this edition to be reviewed.</p>
```